packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -47,7 +47,9 @@
 
                 <form-selector />
 
-                <user-note data-test-id="user-note" />
+                <user-note
+                    data-test-id="user-note"
+                    @input="updateUserNote($event.target.value)" />
```

Comment:
Does this work without `.native`?

Reply:
Yes, it seems to! 😄

Reply:
Weird because user-note is wrapped in a div so it should silently braik with listening to the div